tests/smc/test_smc.py

```diff
@@ -28,6 +29,8 @@
 from pymc.smc.kernels import IMH, systematic_resampling
 from tests.helpers import assert_random_state_equal
 
+_IS_WINDOWS = platform.system() == "Windows"
```

Comment:
@aloctavodia I had to disable the progressbar on windows. Tests were failing with a cryptic error that I feel is unrelated?

https://github.com/pymc-devs/pymc/pull/6991#issuecomment-1795066441

I couldn't get on a Windows machine to reproduce.